app/src/main/java/com/example/trainingintive/what_to_do_feature/presentation/ActivityModelRxAdapter.kt

```diff
@@ -29,8 +24,27 @@ class ActivityModelAdapter @Inject constructor() : RecyclerView.Adapter<Activity
     }
 
     override fun onBindViewHolder(holder: ItemViewHolder, position: Int) {
-        holder.textView.text = activities[position].activity
+        getItem(position)?.let {
+            holder.textView.text = it.activity
+        }
     }
 
-    override fun getItemCount(): Int = activities.size
+    class ItemViewHolder(view: View) : RecyclerView.ViewHolder(view) {
+        val textView: TextView = view.findViewById(R.id.item_activity)
+    }
+
+    companion object {
+        private val COMPARATOR = object : DiffUtil.ItemCallback<ActivityModel>() {
```

Comment:
Why this val name is in uppercase :D?

Reply:
And what is the purpose to use `companion object` here? Maybe it would be simpler to just create a Comparator object instead of creating val in a companion object?